framework/BaseForm.js

```diff
@@ -0,0 +1,23 @@
+import Label from "./elements/Label";
+
+
+class BaseForm {
+
+    constructor(locator, name) {
+
+        this.locator = locator;
+        this.name = name;
+    }
+
+    async isFormDisplayed() {
+        let isFormDisplayed;
+        let labelElement = await new Label(this.locator, this.name);
+        try {
```

Comment:
i believe it won't throw an error since you've already have this try catch in isDisplayed